client/wildcard/src/components/Tabs/Tabs.tsx

```diff
@@ -88,13 +102,19 @@ export const Tabs = React.forwardRef((props, reference) => {
 
 export const TabList = React.forwardRef((props, reference) => {
     const { actions, as = 'div', wrapperClassName, className, ...reachProps } = props
+    const { longTabList } = useTabsSettings()
+
     return (
         <div className={classNames(styles.tablistWrapper, wrapperClassName)}>
             <ReachTabList
                 data-testid="wildcard-tab-list"
                 as={as}
                 ref={reference}
-                className={classNames(className, styles.tabList)}
+                className={classNames(
+                    className,
+                    styles.tabList,
+                    longTabList === 'scroll' ? styles.tabListScroll : undefined
```

Comment:
To be consistent with the approach used in the codebase. We rely on the fact that the `classNames` utility handles falsy values for us. 

```suggestion
                    longTabList === 'scroll' && styles.tabListScroll
```

Reply:
Thank you, fixed.